pkg/cluster_upgrader/cluster_upgrader.go

```diff
@@ -715,29 +736,29 @@ func ValidateUpgradeConfig(c client.Client, m maintenance.Maintenance, upgradeCo
 
 	//Get current version, then compare
 	current := getCurrentVersion(clusterVersion)
-	log.Info(fmt.Sprintf("current version is %s", current))
+	logger.Info(fmt.Sprintf("current version is %s", current))
 	if len(current) == 0 {
 		return false, fmt.Errorf("failed to get current version")
 	}
 	// If the version match, it means it's already upgraded or at least control plane is upgraded.
 	if current == upgradeConfig.Spec.Desired.Version {
-		log.Info("the expected version match current version")
+		logger.Info("the expected version match current version")
 		metrics.UpdateMetricValidationFailed(upgradeConfig.Name)
 		return false, fmt.Errorf("cluster is already on version %s", current)
 	}
 
 	// Compare the versions, if the current version is greater than desired, failed the validation, we don't support version rollback
 	versions := []string{current, upgradeConfig.Spec.Desired.Version}
-	log.Info("compare two versions")
+	logger.Info("compare two versions")
 	sort.Strings(versions)
 	if versions[0] != current {
-		log.Info(fmt.Sprintf("validation failed, current version %s is greater than desired %s", current, upgradeConfig.Spec.Desired.Version))
+		logger.Info(fmt.Sprintf("validation failed, current version %s is greater than desired %s", current, upgradeConfig.Spec.Desired.Version))
 		metrics.UpdateMetricValidationFailed(upgradeConfig.Name)
 		return false, fmt.Errorf("desired version %s is greater than current version %s", upgradeConfig.Spec.Desired.Version, current)
 	}
 
 	// Find the available versions from cincinnati
-	uuid, err := uuid.Parse(string(clusterVersion.Spec.ClusterID))
+	uuID, err := uuid.Parse(string(clusterVersion.Spec.ClusterID))
```

Comment:
_Extremely_ minor and pedantic nit - would prefer this stay as uuid - it's not really a convention to have the mixed case for the term.

Reply:
This var clashes with the package name and my IDE suggested to fix it. maybe `clusterId`?

Reply:
clusterId sounds good :+1: